tutorials/v7/draw_mt.cxx

```diff
@@ -53,25 +57,43 @@ void draw_canvas(const std::string &title, RColor col)
    }
 
    // Create a canvas to be displayed.
-   auto canvas = RCanvas::Create(title);
+   auto canvas = RCanvas::Create(title + " canvas");
    canvas->Draw(pHist)->SetLineColor(col);
+   // canvas->Draw(pHist)->SetLineColor(RColor::kRed);
    canvas->Draw(pHist2)->SetLineColor(RColor::kBlue);
 
-   canvas->Show();
 
-   for (int loop=0;loop<50;++loop) {
+   bool batch = gROOT->IsWebDisplayBatch();
+
+   int maxloop = batch ? 10 : 100;
+
+   if (title == "Zero") {
```

Comment:
Remove? ("Zero" seems to never be the title.)

Reply:
Yes, it was try to solve I/O problem